packages/spindle-ui/src/NavigationTab/UnderlineTab/UnderlineTab.figma.tsx

```diff
@@ -6,7 +6,10 @@ figma.connect(
   UnderlineTab,
   'https://www.figma.com/design/FSgvRthUiMMXWgrSE4RUgr/Spindle-UI?node-id=4053-27265&t=YLIDc1qkFqK1kafo-4',
   {
-    imports: ["import { UnderlineTab } from '@openameba/spindle-ui';"],
+    imports: [
+      "import { UnderlineTab } from '@openameba/spindle-ui';",
+      "import '@openameba/spindle-ui/NavigationTab/UnderlineTab/UnderlineTab.css';",
```

Comment:
```suggestion
      "import '@openameba/spindle-ui/NavigationTab/index.css';",
```

すみません。今更気づいたんですが、こちらが正しいかもしれません。

Reply:
現状っていうのはどこでだっけ..!?
てのとファイルはこりかな？
https://github.com/openameba/spindle/blob/main/packages/spindle-ui/src/NavigationTab/UnderlineTab/UnderlineTab.css

Reply:
すみません！言葉足らずでした。distに出力されるファイルに含まれておらず、index.cssとして出力されているという意味合いでした。
<img width="800" alt="image" src="https://github.com/user-attachments/assets/6b5e5808-11e7-49f6-abdf-1569ad335d4f" />

Reply:
あーなるほど、てことは今のStorybookの記述も違うんですな
```

Reply:
そうなんです。
最終的に出力側を修正する説濃厚なんですが、そのタイミングが来るまでは現状出力されてるpathを参照するのが良いかなと。

Reply:
ありがとうございます:pray: